main.py

```diff
@@ -0,0 +1,44 @@
+import ssl
+import os
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from datetime import datetime, timedelta
+from urllib.parse import urlparse
+import telebot
+
+# access token for telegram bot
+access_token = os.environ.get('API_KEY')
+# the time to notify of the expiration of the certificate(days)
+notice_date = 10
+bot = telebot.TeleBot(access_token)
+
+
+def check_certificate(url: str):
+    parsed_url = urlparse(url)
+    try:
+        cert = ssl.get_server_certificate(
+            (parsed_url.hostname, parsed_url.port or 443))
+    except Exception:
```

Comment:
What the reason to use try / except statement here? The except block does nothing. And also it's a bad practise to except base Exception, if you want to catch an error you must be as specific as it possible